unittests/tools/test_fortify_parser.py

```diff
@@ -75,3 +74,14 @@ def test_fortify_issue6082(self):
             self.assertEqual("src/adrui/MyContextListener.java", finding.file_path)
             self.assertEqual(28, finding.line)
             self.assertEqual('B5B15F27E10F4D7799BD0ED1E6D34C5D', finding.unique_id_from_tool)
+
+    def test_fortify_many_fdr_findings(self):
+        testfile = open("unittests/scans/fortify/many_findings.fpr")
+        parser = FortifyParser()
+        findings = parser.get_findings(testfile, Test())
+        self.assertEqual(61, len(findings))
+        with self.subTest(i=0):
+            finding = findings[0]
+            self.assertEqual("Cross-Site Request Forgery 114E5A67-3446-4DD5-B578-D0E6FDBB304E", finding.title)
+            self.assertEqual("High", finding.severity)
+            self.assertEqual('114E5A67-3446-4DD5-B578-D0E6FDBB304E', finding.unique_id_from_tool)
```

Comment:
this is actually the class id and not the finding id

Reply:
This check is right I would say.